### ISSUE TITLE
exponential lr

### ISSUE THREAD

**Issue by Robert-Hopkins** [wizyoung/YOLOv3_TensorFlow#219]
as epoch goes up， why does the lr increase instead of decrease？
such as：
3.1095e-06
6.2189e-06
9.3284e-06
1.2438e-05
1.5547e-05
...

**Comment by aaron38** [comment]
Do you use the warmup strategy for training? During the warmup epochs, the learning rate gradually increases, to prevent the gradient from exploding. After the warmup, the lr should stay at the same value
